clouddriver-eureka/README.md

```diff
@@ -37,3 +37,8 @@ Each account definition in AWS must also define a `discovery` field that denotes
       regions:
         - name: us-east-1
 ```
+
+By default, only one Eureka is supported per AWS account. If you have multiple 
+`aws.accounts` configured in clouddriver which share an accountId, and wish to
+use a separate Eureka for each, then you can enable support by setting
+`eureka.provider.allowMultipleEurekaPerAccount` to true.
```

Comment:
Good ideas--I've added an example and mentioned that this won't work with a different cloud provider or with the titus integration like previously discussed.